detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -12,10 +14,6 @@ namespace Synopsys.Detect.Nuget.Inspector.Inspection.Inspectors
 {
     class SolutionInspector : IInspector
     {
-        private List<string> ExcludedProjectTypeGUIDs = new List<string>() {
```

Comment:
Removing this, since we would need to scan every folder now for checking if we have a Directory.Packages.props file to parse it.